examples/scale_bar.py

```diff
@@ -21,7 +21,22 @@
     scale=(0.29, 0.26, 0.26),
 )
 viewer.scale_bar.visible = True
-viewer.scale_bar.unit = 'um'
+
+# Text options
+viewer.scale_bar.unit = 'um' # set to None to diplay no unit
+viewer.scale_bar.length = 23 # length, in units, of the scale bar
+viewer.scale_bar.font_size = 20 # default is 10
+
+# Text color
+viewer.scale_bar.colored = True # default value is False
+viewer.scale_bar.color = 'yellow' # default value is magenta: (1,0,1,1)
+
+# Background box
+viewer.scale_bar.box = True # add background box, default is False
+viewer.scale_bar.box_color = (1, 0, 1, 0.4) # this is magenta with alpha=0.4
+
+# Scale bar position
+viewer.scale_bar.position = 'bottom_left' # default is 'bottom_right'
```

Comment:
Minor nitpick: PEP-8 [specifies](https://pep8.org/#inline-comments) two spaces after the code for inline statements:

> Inline comments should be separated by at least two spaces from the statement.

Reply:
This encourages me to double check I have linting set up correctly in that environment, thank you!